frontend/keyboardNav.js

```diff
@@ -43,6 +43,17 @@ module.exports = function keyboardNav(store: Store, win: Object): (e: DOMEvent)
       return;
     }
     e.preventDefault();
+    if ((e.altKey && direction === 'right') || (e.ctrlKey && e.altKey && direction === 'right')) {
```

Comment:
Isn't the second condition extraneous? It seems to me that the left is always true when the right is true.
Same below.